### ISSUE TITLE
Consider returning dataclasses

### ISSUE THREAD

**Issue by cmungall** [cthoyt/ontoportal-client#3]
Currently the only method `get_ontologies()` doesn't have any return type annotated - it just passes directly to get_json which returns a dict

https://ontoportal-client.readthedocs.io/en/latest/_modules/ontoportal_client/api.html#OntoPortalClient.get_json

Consider returning either a typed dict or dataclasses/pydantic

This would obviously require a tighter coupling but I think this is a good thing - at the moment there is a not a massive advantage to using this client vs raw requests

**Comment by cthoyt** [comment]
Yes, I think there should be a dichotomy of low-level operations that just get the results back then on top of that we can implement operations that do some processing and wrapping.
